### PR TITLE
Split network response completed tests

### PR DESCRIPTION
**Pull request by thiagowfx**
Move status=101 and status=407 to a separate file.

This is because they cannot yet be individually disabled (#42482).

### CODE REVIEW

**Review by Lightning00Blade — APPROVED** [review]

**Review by OrKoN — COMMENTED** [review]
It looks like the tests are flaky?

**Comment by thiagowfx** [comment]
Could someone from Mozilla also take a look?

File: webdriver/tests/bidi/network/response_completed/response_completed_tentative.py
Comment (whimboo) [inline webdriver/tests/bidi/network/response_completed/response_completed_tentative.py:10]: Is there a GitHub url that we could add here which explains why this is tentative?

Comment (thiagowfx) [reply webdriver/tests/bidi/network/response_completed/response_completed_tentative.py:10]: The reason why it is tentative is that it is not working on Chromium.
"Not working" is typically not a big deal because we can just disable the test / mark expectations as failing, however this cannot be done in this scenario due to https://github.com/web-platform-tests/wpt/issues/42482.

Should I add a comment in the file to say it is blocked on https://github.com/web-platform-tests/wpt/issues/42482?

There's no problem with the tests themselves - my goal is only to unblock our development / CI by disabling two subtests. This is the only workaround I could find - split them into another file.

Comment (juliandescottes) [reply webdriver/tests/bidi/network/response_completed/response_completed_tentative.py:10]: I think we used "tentative" for things still in flux spec-wise (eg spec PR not merged yet). It's fine to split, but maybe find another name? eg `response_completed_status.py`, still with a comment explaining why we had to split this.

Comment (whimboo) [reply webdriver/tests/bidi/network/response_completed/response_completed_tentative.py:10]: @jgraham do you have some feedback in why we do not yet support `label` based on a `subtest` metadata entries? I mean there is an issue now but would be good to know why we didn't consider that yet. Thanks.

Comment (thiagowfx) [reply webdriver/tests/bidi/network/response_completed/response_completed_tentative.py]: Done

**Review by whimboo — APPROVED** [review]

**Comment by thiagowfx** [comment]
@DanielRyanSmith could you please admin merge? Thanks

**Pull request merged**
